clog/readers.py

```diff
@@ -176,10 +178,19 @@ def __str__(self):
     def __repr__(self):
         return "<StreamTailerSetupError host=%r port=%r message=%r>" % (self.host, self.port, self.message)
 
+
 def get_settings(setting):
-    with open(SETTINGS_FILE) as settings_file:
-        settings = load(settings_file)
-    return settings[setting]
+    """Reads the specified setting from one of the files listed in the
+    SETTING_FILES variable. The first existing file will be used.
+    If no file in the list exist, an exception will be thrown."""
+    for file_path in SETTINGS_FILES:
+        try:
```

Comment:
I think this complexity is unwarranted.
We have strong controls over our infrastructure and can ensure this file is either in the right place or has a symlink to reference it.